### PR TITLE
Implement loans page

### PR DESCRIPTION
**Pull request by evgenyboxer**
@justinjmoses @clementbalestrat this PR is going to be pretty big, so it's probably best to review it by commits. Early feedback is welcomed (this is still a work in progress)

My goal is to use translations / stories and best practices for the new code (and refactor existing one if needed)

- [x] Implement header links
- [x] Extract common components from the trade page (put it in Root / commonStyles)
- [x] Implement routing and initial boilerplate for Loans page
- [ ] Implement "empty boxes" for all the different sections
- [ ] Implement eth collateral dashboard (UI only)
- [ ] Implement create loan (UI only)
- [ ] Implement "my loans" (UI only) using react-table
- [ ] Hook data
- [ ] Add loaders where needed
- [ ] Implement header "new" feature
- [ ] Implement trade "borrow or buy" banner in pair list

Fixes #113

### CODE REVIEW

File: src/shared/commonStyles.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export const linkCSS = css`
+	text-decoration: none;
+	&:hover {
+		text-decoration: none;
+	}
+`;
+
+export const ExternalLink = styled.a.attrs({
+	target: '_blank',
+	rel: 'noopener',
```
Comment (evgenyboxer) [inline src/shared/commonStyles.js]: https://developers.google.com/web/tools/lighthouse/audits/noopener

File: src/pages/Root/Root.js [hunk 2]
```diff
@@ -91,20 +97,20 @@ const Root = ({
...
 						<Switch>
-							<Route path="/trade">
-								<Trade />
-							</Route>
-							<Route path="/">
-								<Trade />
-							</Route>
+							<Route path={ROUTES.Trade} component={Trade} />
+							<Route path={ROUTES.Loans} component={Loans} />
+							<Redirect to={ROUTES.Trade} />
```
Comment (evgenyboxer) [inline src/pages/Root/Root.js]: When users land on the page they will be redirected to `/trade`. Let me know if its OK (this helps with `NavLink` detect that we are on the exchange).

Comment (clementbalestrat) [reply src/pages/Root/Root.js]: yeah I'm happy with that!

**Comment by evgenyboxer** [comment]
To keep conflicts low, it was decided to send smaller PR's which will be merged frequently (provided they don't break anything)

**Pull request merged**